pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorSubscriptionTest.java

```diff
@@ -868,6 +869,7 @@ public void testReplicatedSubscriptionWithCompaction() throws Exception {
                 .topic(topicName)
                 .subscriptionName("sub2")
                 .subscriptionType(SubscriptionType.Exclusive)
+                .subscriptionInitialPosition(SubscriptionInitialPosition.Earliest)
```

Comment:
Is it reflect the PR's description `This change will also make subscriptionInitialPosition=Latest work normally since the mark-delete doesn't is null.` ?

Reply:
Yes, consumer always be read from earliest position after this PR.

Reply:
Ok, I see. So after this change, the consumer will start to read from the latest if it doesn't have `SubscriptionInitialPosition.Earliest`

Reply:
Yesh, `SubscriptionInitialPosition` will work normally